File: Data Structures & Algorithms/remove-duplicates-from-sorted-array/submission-0.cs
public class Solution {
    public int RemoveDuplicates(int[] nums) {
        int i = 0, j = 1;
        while(j != nums.Length) {
             if (nums[j] != nums[i]) {
                nums[i+1]=nums[j];
                i++;
             }
             j++;
        }
        return i+1;
    }
}

File: Data Structures & Algorithms/duplicate-integer/submission-0.py
# If duplicate is found, it doesnt process further. 
# Better than len(nums) and len(set(nums))

class Solution:
    def hasDuplicate(self, nums: List[int]) -> bool:
        hashSet = set()
        for num in nums:
            if num in hashSet:
                return True
            else:
                hashSet.add(num)
        return False
